src/util.cc

```diff
@@ -47,7 +47,8 @@ TwoByteValue::TwoByteValue(Isolate* isolate, Local<Value> value)
     return;
 
   // Allocate enough space to include the null terminator
-  size_t len = StringBytes::StorageSize(isolate, string, UCS2) + 1;
+  size_t len = StringBytes::StorageSize(isolate, string, UCS2) +
+    sizeof(uint16_t);
```

Comment:
Break right after the '=' and indent the next line by four spaces.  Otherwise LGTM and good catch.